json/src/vault.rs

```diff
@@ -170,3 +184,11 @@ pub struct VaultEstimation {
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct TokenPercentageSplit(HashMap<String, u64>);
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum VaultResult {
+    VaultActive(VaultActive), // Verbose active
+    VaultLiquidation(VaultLiquidation), // Verbose in liquidation
+                              // Vault(Vault),                       // Any state non-verbose
```

Comment:
Why is this commented ? I think one of `getvault` or `listvaults` uses this variant in its non-verbose response

Reply:
huh... hmm.. i didn't change that section 🤔  .. just did a `merge main`... 
https://github.com/DeFiCh/rust-defichain-rpc/commit/89aeeeb101de33e8e3849c8783e82ac1dcfce3e2